codegenerator/cli/templates/static/codegen/src/eventFetching/ChainManager.res

```diff
@@ -306,13 +297,12 @@ let rec createBatchInternal = (
         )
       | EventFetchers(item, fetchStatesMap) => (arbitraryEventQueue, fetchStatesMap, item)
       }
+      let _ = batch->Js.Array2.push(nextItem)
```

Comment:
Is it ok to push here?

Reply:
Yeah "batch" is internal to this function.